### PR TITLE
V0.1.7.post0

### PR DESCRIPTION
**Pull request by adament**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+  url: https://github.com/osqp/qdldl-python/archive/refs/tags/v.{{ version }}.tar.gz
+  sha256: 0a58b0a44deafb7cfb89dc00b3b28a45b1f9da9d3f486503675476f8082c4ef3
   patches:
-    - patches/0001-use-Ninja-everywhere.patch
-    - patches/0002-build-against-conda-forge-s-libqdldl.patch
-    - patches/0003-build-qdldlamd-as-shared-lib-and-install-it.patch
+    - patches/0001-Do-not-pass-CMAKE_GENERATOR-as-variable.patch
+    - patches/0002-use-Ninja-everywhere.patch
+    - patches/0003-build-against-conda-forge-s-libqdldl.patch
+    - patches/0004-build-qdldlamd-as-shared-lib-and-install-it.patch
```
Comment (h-vetinari) [inline recipe/meta.yaml]: Could you add your patch at the end (should be easy to do if you do an interactive rebase on the branch that generated these patches, and order your patch last)?

File: recipe/patches/0001-Do-not-pass-CMAKE_GENERATOR-as-variable.patch [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ 
+ # Add parameters to cmake_args and define_macros
+-cmake_args += ["-DUNITTESTS=OFF"]
++cmake_args = ["-DUNITTESTS=OFF"]
+ cmake_build_flags = []
+ lib_subdir = []
+ 
+-- 
+2.43.0.windows.1
+
```
Comment (h-vetinari) [inline recipe/patches/0001-Do-not-pass-CMAKE_GENERATOR-as-variable.patch]: When generating the patches, please use `git format-patch <hash> --no-signature`

**Review by h-vetinari — COMMENTED** [review]
Thanks for the PR! Just some minor things. Also, feel free to add yourself as a maintainer if you'd like!

File: recipe/meta.yaml (hunk 1 above)
Comment (adament) [reply recipe/meta.yaml]: Done, though I still ran format patch on the entire patch set, would you prefer I take your patch files from the 0.1.7 branch?

File: recipe/patches/0001-Do-not-pass-CMAKE_GENERATOR-as-variable.patch (hunk 2 above)
Comment (adament) [reply recipe/patches/0001-Do-not-pass-CMAKE_GENERATOR-as-variable.patch]: Thank you, hopefully I managed to do it right this time.

**Review by h-vetinari — APPROVED** [review]
Thanks a lot! :)

**Pull request merged**